packages/nimble-components/CONTRIBUTING.md

```diff
@@ -41,9 +39,11 @@ Before building a new component, 3 specification documents need to be created:
 ## Development workflow
 
 1. When creating new components, create the folder structure and decide how to implement the component as described in [Develop new components](#develop-new-components).
-2. Run the Storybook command from the `nimble` directory:
+2. Run the Storybook build and start commands from the `storybook` package directory:
```

Comment:
Anyone else interested in having a top-level Storybook NPM script, so I don't need to `cd` into the Storybook directory? I.e. `"storybook": "npm run start --prefix packages/storybook"` from top-level `package.json`

Reply:
Should be considered in a follow-up PR. I'd be interested in it handling both the `component watch` (potentially both spright and nimble components) and `storybook start` to be a really handy shortcut. Probably leveraging concurrently

Reply:
@rajsite any objection to making Fred's dream come true?